scripts/zones/Dynamis-Beaucedine/mobs/BeZhe_Keeprazer.lua

```diff
@@ -0,0 +1,14 @@
+-----------------------------------
+-- Area: Dynamis Beaucedine
+--  MOB: BeZhe_Keeprazer
+-----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setLocalVar("mainSpec", dsp.jsa.ASTRAL_FLOW_MAAT)
```

Comment:
not sure if right animation for these but I think thats a problem that predates this pr